ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -3,27 +3,29 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#if (and this.version.hasSecretsSync (not @isActivated))}}
-  {{#unless this.hideOptIn}}
-    <Hds::Alert
-      @type="inline"
-      @color="warning"
-      @onDismiss={{fn (mut this.hideOptIn) true}}
-      data-test-secrets-sync-opt-in-banner
-      as |A|
-    >
-      <A.Title>Enable Secrets Sync feature</A.Title>
-      <A.Description>To use this feature, specific activation is required. Please review the feature documentation and enable
-        it. If you're upgrading from beta, your previous data will be accessible after activation.</A.Description>
-      <A.Button
-        @text="Enable"
-        @color="secondary"
-        {{on "click" (fn (mut this.showActivateSecretsSyncModal) true)}}
-        data-test-secrets-sync-opt-in-banner-enable
-      />
-    </Hds::Alert>
-  {{/unless}}
-{{/if}}
+{{#unless @isActivated}}
+  {{#if (or @licenseHasSecretsSync @isManaged)}}
```

Comment:
Instead of wrapping it in the additional `{{#unless}}` is it possible to add this conditional to the beginning of the `{{#if}}` statement? It makes it a little easier to parse the logic (in my opinion), rather than having to switch from unless, to if, then back to unless 
```hbs
{{#if (or (not @isActivated) @licenseHasSecretsSync @isManaged)}}

Reply:
Our brains are oil and water sometimes! For me it read easier to read the first conditional: e.g. "okay, this only applies to not activated and then I check the next set of conditionals". But I'll defer to your suggestion as it's one less unless to deal with.

Reply:
Update: I ended up keeping as is. I tried the one line and then when updating a test it scrambled my brain to parse through the logic. 🍳